test/test_data/with_tuple_target.py

```diff
@@ -0,0 +1,3 @@
+with a as (x, y):
```

Comment:
I will add more tests that check the recursive case. For example:

```
with a as ((x,y), z):  pass
```

Reply:
I was thinking that maybe `del_tuple.py` was enough, but still better to be sure.

Reply:
Will do a complete review tomorrow! Today is late :)

Reply:
Agreed!

Reply:
> I was thinking that maybe `del_tuple.py` was enough, but still better safe than sorry.

Indeed, ideally we should have test cases for all possibilities to avoid regression due to potential refactors.